airflow/api_fastapi/core_api/security.py

```diff
@@ -82,6 +82,23 @@ def callback():
     return inner
 
 
+def requires_access_asset(method: ResourceMethod) -> Callable:
+    def inner(
+        uri: str | None = None,
+        user: Annotated[BaseUser | None, Depends(get_user)] = None,
+    ) -> None:
+        def callback():
+            return get_auth_manager().is_authorized_asset(
+                method=method, details=AssetDetails(uri=uri), user=user
+            )
+
+        _requires_access(
+            is_authorized_callback=callback,
```

Comment:
Something I don't understand, where is handled the part where `uri` is `None` and therefore we need access on 'all' assets ?

Reply:
I made it similar to what was written in api_connexion. Should I be specifically handling the case when uri=None?

Reply:
I think it's ok for now